### PR TITLE
#784 Nightly release and release without mpi

### PR DESCRIPTION
**Pull request by barondim**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non regression tests were added (update of inputs, outputs, SA, algos)
- [ ] main documentation was updated (update of input/output file)
- [x] the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Comment by gautierbureau** [comment]
we don't need to do it for Linux release

**Comment by barondim** [comment]
> we don't need to do it for Linux release

On the dyn-algo task https://github.com/dynawo/dynawo-algorithms/pull/406 we build dyn-algo with and without MPI. Isn't it coherent to do the same with dfl ?

**Review by gautierbureau — APPROVED** [review]

File: sources/main.cpp [hunk 1]
```diff
@@ -310,7 +310,9 @@ int main(int argc, char *argv[]) {
 
     // NSA: wait so that the steady state computation is over for everyone and the results written by root process
     DYNAlgorithms::multiprocessing::Context::sync();
+#ifdef _MPI_
```
Comment (gautierbureau) [inline sources/main.cpp:313]: should be encapsulated in the context I think

**Review by gautierbureau — CHANGES_REQUESTED** [review]

**Comment by barondim** [comment]
- [ ] to be merged with https://github.com/dynawo/dynawo-algorithms/pull/441

**Comment by barondim** [comment]
- [ ] to be merged with https://github.com/dynawo/dynawo-algorithms/pull/442

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**